app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\User;
+
+class UserController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(User $user)
+    {
+        //
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        //
+        $user = User::find($id);
```

Comment:
You can create a user Instance directly by typecastig `$id` to `User` so it will be 

```
puclic function show(User $id) {
...
}
```

to make it more relevant you can change your parameter name from `$id` to just `$user`